<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPClass</class>
 <widget class="QMainWindow" name="DPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>654</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>DP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QComboBox" name="cbx_device"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_show_pic">
          <property name="text">
           <string>无</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>初始化</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_open">
             <property name="text">
              <string>打开设备</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_close">
             <property name="text">
              <string>关闭设备</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="btn_search">
             <property name="text">
              <string>连接设备</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>图像采集</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_start_grabing">
             <property name="text">
              <string>开始采集</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_end_grabing">
             <property name="text">
              <string>结束采集</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_exposure">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string> 曝光</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="let_exposure"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_gain">
             <property name="text">
              <string> 增益</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="let_gain"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="btn_set_param">
             <property name="text">
              <string>设置参数</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="btn_get_param">
             <property name="text">
              <string>获取参数</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>图片保存</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_save_BMP">
             <property name="text">
              <string>保存BMP</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_save_JPG">
             <property name="text">
              <string>保存JPG</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>相机状态</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_state">
             <property name="text">
              <string>无</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>应用</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>查看保存的图片</string>
     </property>
     <addaction name="actionBMP"/>
     <addaction name="actionJPG"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="action1"/>
    <addaction name="action2"/>
    <addaction name="action3"/>
    <addaction name="menu_2"/>
    <addaction name="actionip"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionwen">
   <property name="text">
    <string>wen</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>运动控制</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>FR_IQA</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>主观评价</string>
   </property>
  </action>
  <action name="actionBMP">
   <property name="text">
    <string>BMP</string>
   </property>
  </action>
  <action name="actionJPG">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>设置ip地址</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>设置相机IP</string>
   </property>
  </action>
  <action name="action4_2">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>电机</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="text">
    <string>相机</string>
   </property>
  </action>
  <action name="actionip">
   <property name="text">
    <string>设置IP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DP.qrc"/>
 </resources>
 <connections/>
</ui>
